src/main/java/io/github/thebusybiscuit/slimefun4/api/events/ClimbingPickLaunchEvent.java

```diff
@@ -0,0 +1,84 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.Vector;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.tools.ClimbingPick;
+
+/**
+ * An {@link Event} that is called whenever a {@link Player} has
+ * used a {@link ClimbingPick} on a climbable surface.
+ *
+ * @author Linox
+ *
+ * @see ClimbingPick
+ *
+ */
+public class ClimbingPickLaunchEvent extends Event implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private final Player player;
+    private final Vector velocity;
+    private final ItemStack pick;
+    private boolean cancelled;
+
+    public ClimbingPickLaunchEvent(Player player, Vector velocity, ItemStack pick) {
```

Comment:
I very much like the Addition of this Event but it should also include a reference to the corresponding `ClimbingPick` that was used, in case there may be multiple kinds in the future.